extension/src/popup/views/__tests__/SignTransaction.test.tsx

```diff
@@ -359,7 +363,7 @@ describe("SignTransactions", () => {
     );
   });
 
-  it("memo: render memo return", async () => {
+  xit("memo: render memo return", async () => {
```

Comment:
I'm still confused as to why these tests are failing - the test runner is saying it's because Horizon is returning a 405 (too many requests), but this view doesn't make any Horizon requests. I only get this error when I run all the tests together. Running these tests individually works as expected.

My theory is that some other tests are actually hitting Horizon a bunch, prompting the 405, and the error is surfacing here?